### PR TITLE
Add Search schema known limitation for Graph Search API 

### PR DESCRIPTION
**Pull request by yiwenwang-ms**

### CODE REVIEW

File: api-reference/v1.0/resources/search-api-overview.md [hunk 1]
```diff
@@ -205,6 +205,8 @@ The search API has the following limitations:
 
 - Customizations in SharePoint search, such as a custom search schema or result sources, can interfere with Microsoft Search API operations.
 
+- Graph Search API doesn't support site-level [search schema](/sharepoint/manage-search-schema)
```
Comment (njerigrevious) [inline api-reference/v1.0/resources/search-api-overview.md:208]: Can we make this clearer to say "Graph Search API doesn't support site-level search schema, only tenant-level"?

**Review by njerigrevious — CHANGES_REQUESTED** [review]
Looks good but can we make the clarification a bit clearer on what schema is supported for managd properties?

File: api-reference/beta/resources/search-api-overview.md [hunk 2]
```diff
@@ -221,6 +221,8 @@ The search API has the following limitations:
 
 - Customizations in SharePoint search, such as a custom search schema or result sources, can interfere with Microsoft Search API operations.
 
+- Graph Search API doesn't support site-level [search schema](/sharepoint/manage-search-schema)
```
Comment (Lauragra) [inline api-reference/beta/resources/search-api-overview.md:224]: Global: Please revise the text as follows:

The search API doesn't support the site-level [search schema](/sharepoint/manage-search-schema).

**Review by njerigrevious — APPROVED** [review]
looks good - thank you!

**Pull request merged**